### PR TITLE
Fix issues in dxl.xsd that didn't meet XSD standard

### PR DESCRIPTION
**Pull request by MoZhonghua**
I try to generate DXL parser according dxl.xsd in golang, and find some non-compliant issues in this file:
- complexContent should only include (restriction|extension)
- SubPlanTestExpr:  add "dxl:" space prefix

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@MoZhonghua Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=greenplum-db/gporca&pullRequestId=326)!

[Click here](https://cla.pivotal.io/sync/pivotal?repositoryId=greenplum-db/gporca&pullRequestId=326) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@MoZhonghua Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=greenplum-db/gporca&pullRequestId=326)!

File: server/dxl.xsd [hunk 1]
```diff
@@ -1191,7 +1188,7 @@
 		</xsd:complexContent>	
 	</xsd:complexType>
 
-	<xsd:complexType name="SubPlanTestExpr">
+	<xsd:complexType name="dxl:SubPlanTestExpr">
 		<xsd:sequence>
```
Comment (dhanashreek89) [inline server/dxl.xsd]: Could you please let us know why the complexType name here needs to be prefixed with `dxl` ?
Wouldn't the change at line 1465 below be enough ?

**Comment by dhanashreek89** [comment]
@MoZhonghua The PR looks good except for one comment I posted inline. Once that is answered; we will take care of getting this in.

Thank you for your contribution!

File: server/dxl.xsd (hunk 1 above)
Comment (MoZhonghua) [reply server/dxl.xsd]: Oh, you are right!

**Pull request merged**